api/src/booking/booking.service.ts

```diff
@@ -147,73 +147,68 @@ export class BookingService {
     const worksheet = workbook.getWorksheet(1);
     const setCell = setCellHelper(worksheet); // taking advantage of "closure"
 
-    // A10 Last name + First Name
-    setCell({ row: 10, column: 'A', value: `${interpreter.firstName} ${interpreter.lastName}` });
-
-    // A15 Phone
-    setCell({ row: 15, column: 'A', value: interpreter.phone });
-
-    // G15 Email
-    setCell({ row: 15, column: 'G', value: interpreter.email });
+    // B9 Last name + First Name
+    setCell({
+      row: 9,
+      column: 'B',
+      value: concat([interpreter.firstName, interpreter.lastName], ', ', (str: string) => str.toUpperCase()),
+    });
 
-    // B16, E16, H16, K16 Language Level
+    // R9 Language Level
     const bookLang = booking.language;
     const intpLang = interpreter.languages.find(lan => lan.language.name === bookLang.name);
+    setCell({ row: 9, column: 'R', value: String(intpLang.level) });
 
-    switch (intpLang?.level) {
-      case Level.one: {
-        setCell({ row: 16, column: 'B', value: 'X' });
-        break;
-      }
-      case Level.two: {
-        setCell({ row: 16, column: 'E', value: 'X' });
-        break;
-      }
-      case Level.three: {
-        setCell({ row: 16, column: 'H', value: 'X' });
-        break;
-      }
-      case Level.four: {
-        setCell({ row: 16, column: 'K', value: 'X' });
-        break;
-      }
-      default: {
-        setCell({ row: 16, column: 'B', value: 'X' });
-      }
-    }
+    // B11 address + city + province + postcode
+    setCell({ row: 11, column: 'B', value: concat([interpreter.address, interpreter.city, interpreter.province]) });
```

Comment:
Does this not need spaces in between? I think with this you will get '920 StreetVictoriaBC'.

Reply:
I have now read the `concat` function. I think this is an issue with the name of this function.

Reply:
rename to `mapAndJoin`